cpp/oneapi/dal/algo/subgraph_isomorphism/backend/cpu/graph.hpp

```diff
@@ -170,7 +170,8 @@ void graph<Cpu>::init_bit_representation(const dal::preview::detail::topology<st
     bool has_edges_attribute = false;
     std::int64_t vertex_count = t._vertex_count;
     graph_data<Cpu> graph_data_storage;
-    graph_data_storage.pbit_data = new graph_input_bit_data<Cpu>(vertex_count, allocator_);
+    graph_input_bit_data<Cpu>* memory = allocator_.allocate<graph_input_bit_data<Cpu>>(1);
+    graph_data_storage.pbit_data = new (memory) graph_input_bit_data<Cpu>(vertex_count, allocator_);
```

Comment:
If user's allocator or corresponding allocator_traits have 'construct' member function, they both to be ignored.